### DISCUSSION TITLE
Support for disabling xwayland at runtime

### DISCUSSION THREAD

**Discussion by danielfikko** [labwc/labwc#2479] [feature-requests]
Is an option that is present in sway and KDE Plasma, and is a more dynamic way than compiling.

**Comment by danielfikko** [comment]
My account had been flagged as spam because I created two discussions at the same time, so I imagine this discussion was obfuscated. So this message is just bumping, sorry if that's against the rules.

Also, I feel a bit uncomfortable asking for features, because I can't contribute code, but if there's anything else I can do to help this feature progress, I'm willing!

**Comment by NXTler** [comment]
Yeah, this should be a feature. Labwc is mostly used as a wayland compositor from what I can tell, so I don't really see the reason to why it is forced on you.

I know you can disable it on build, but there are not that many people who like building packages from source.

**Comment by charbelnicolas** [comment]
+1, this would be great!

**Comment by danielfikko** [comment]
https://github.com/labwc/labwc/commit/3ce6328f6d18e5035f378b65bbf88217d9d3af26
